servicetalk-dns-discovery-netty/src/main/java/io/servicetalk/dns/discovery/netty/DefaultDnsServiceDiscovererBuilder.java

```diff
@@ -80,6 +90,20 @@ public DefaultDnsServiceDiscovererBuilder minTTL(final int minTTLSeconds) {
         return this;
     }
 
+    /**
+     * The maximum allowed TTL. This will be the maximum poll interval as well as the maximum dns cache value.
+     *
+     * @param maxTTLSeconds the maximum amount of time a cache entry will be considered valid (in seconds).
+     * @return {@code this}.
+     */
+    public DefaultDnsServiceDiscovererBuilder maxTTL(final int maxTTLSeconds) {
```

Comment:
I do not have a strong preference either - the advantage of keeping it separate is that a user who only wants to tweak one is not forced to think about the default value of the other (or we provide public access to the defaults we set). My assumption would be that since you usually construct the builder and then build immediately there won't be really a big difference when the exception is thrown.